src/rejson_api.h

```diff
@@ -55,14 +55,18 @@ typedef struct RedisJSONAPI_V1 {
     int (*getBooleanFromKey)(RedisJSONKey key, const char *path, int *boolean);
 
     // Callee has ownership of `str` - valid until api `close` is called
-    // Return the String or the String representation for Numeric or Bool
-    // FIXME: Handle null field
-    int (*getString)(RedisJSON json, const char **str, size_t *len);
-    int (*getStringFromKey)(RedisJSONKey key, const char *path, const char **str, size_t *len);
+    // Return the String for String
+    // Return the String representation for Numeric
+    // Return true or false for Bool
+    // Return the String representation for Array and Object if strict=0
+    // Return REDISMODULE_ERR for Array and Object if strict=1
+    // Return empty string for null
+    int (*getString)(RedisJSON json, const char **str, size_t *len, int isStrict);
```

Comment:
Why not return a string representation always? The user can check the type if he do not want to do it on array or object no?

Reply:
When `isStrict=0` we always return a string representation.
The user can check the type or can also use `isStrict=1`